src/tests/ftest/slurm_setup.py

```diff
@@ -20,6 +20,8 @@
 PACKAGE_LIST = ["slurm", "slurm-example-configs",
                 "slurm-slurmctld", "slurm-slurmd"]
 
+PACKAGE_VERSION = "18.08.8-1.el7.x86_64"
```

Comment:
Minor: Is it possible to automate it? We will have to change it whenever it is updated.

Reply:
It has recently been updated to version 20 and it broke the install.  However, v20 slurm now has a dependency on hdf5 and they are using an older version (v 1.8 ) and so we would need to rebuild slurm 20 from source in order to use in our env.  slurm 18 is still available.